docs-gen/content/rule_set/data_entry/sensor_actuator.md

```diff
@@ -9,8 +9,9 @@ entry example is given below:
 ```YAML
 Speed:
   type: sensor
-  description: The vehicle speed, as measured by the drivetrain.
-  datatype: uint16
+  description: The vehicle speed.
+  comment: For engine speed see Vehicle.Powertrain.CombustionEngine.Engine.Speed.
```

Comment:
Here I added comment after description. It has no semantic meaning, but I believe our examples should use our preferred order, and I do not know if this is the preferred order.